src/index.ts

```diff
@@ -17,11 +17,27 @@ export const angularModule = angular.module(name, [])
 
         init: function($scope, el): void {
           if (!el.hasAttribute('popgun-listening')) {
-            el.addEventListener('PopgunContentSetup', function(e) {
-              let pop = popgun.getPopFromGroupId((<Element>e.target).getAttribute('popgun-group'));
-              $compile(pop.popOver.element)($scope);
-              $scope.$apply();
-            }, false);
+            el.addEventListener('PopgunContentSetup', (function($compileScope) {
```

Comment:
it feels overcompilcated and hard to read to use an self executing function here, the $scope can be just as easily trapped in closure without it and if the name is the reason we can just do `var $compileScope = $scope;`